Kernel/Arch/x86/x86_64/Processor.cpp

```diff
@@ -42,6 +41,7 @@ asm(
 asm(
 ".global do_assume_context \n"
 "do_assume_context: \n"
+"    int 3 \n"
```

Comment:
is this meant to be here? (amazing work btw :)

Reply:
Thank you! Also, no, but it doesn't work atm either way. I've removed it for now. :)